templates/noKibana/transactions.go

```diff
@@ -7,7 +7,7 @@ var Transactions = Object{
 	},
 	"settings": Object{
 		"number_of_shards":   5,
-		"number_of_replicas": 0,
+		"number_of_replicas": 1,
```

Comment:
shouldn't we change number_of_replicas to 1 on all indexes?

Reply:
no, because one replica means double the storage. Also, the `transactions` index was the only one affected and it is the most used, so it's better to increase the replicas count only for this for now